make/modules/jdk.charsets/Gensrc.gmk

```diff
@@ -29,7 +29,7 @@ include GensrcCommonJdk.gmk
 #
 # Generate files using the charsetmapping tool
 #
-CHARSET_DATA_DIR := $(TOPDIR)/make/data/charsetmapping
+CHARSET_DATA_DIR := $(TOPDIR)/src/java.base/share/data/charsetmapping
```

Comment:
Is it intentional to leave `java.base` literal here, or should it be replaced with `$(MODULE_SRC)`? I see this inconsistency in other tools' `gensrc.gmk` too

Reply:
This is part of the weirdness of charsetmapping that Alan talks about. The charsetmapping data is shared between java.base and jdk.charsets in a way that makes it non-trivial to disentangle. 

So this reference to java.base is quite intentional -- replacing it with $(MODULE_SRC) would have pointed to src/jdk.charsets instead of src/java.base, which would have been incorrect.

Reply:
Thanks for reminding me. Then yes, I'd agree with Alan. It'd be much simpler to exclude this from this PR.